.claude/commands/notebook-review.md

```diff
@@ -1,36 +1,13 @@
 ---
-allowed-tools: Bash(gh pr comment:*),Bash(gh pr diff:*),Bash(gh pr view:*)
+allowed-tools: Bash(gh pr comment:*),Bash(gh pr diff:*),Bash(gh pr view:*),Bash(echo:*),Read,Glob,Grep,WebFetch
```

Comment:
Is `WebFetch` a bit too permissive? Seems like that could expose this to being prompt injected.